src/page/ProfileBuilder.jsx

```diff
@@ -54,6 +54,13 @@ const ProfileBuilder = () => {
     });
   };
 
+  const handleTextInput = (event) => {
+    setFormData({
+      ...formData,
+      [event.target.name]: event.target.value,
+    });
+  };
+
```

Comment:
maybe we could just stick with `handleChange` instead? (is there any difference between using these two functions apart from trimming?)